learn/guides/developing-a-restful-api-with-ballerina.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: developing-a-restful-api-with-ballerina
+title: Developing a RESTful API with Ballerina
+permalink: /learn/developing-a-restful-api-with-ballerina/
+description: This simple guide helps you understand the basics of Ballerina constructs which allow you to write RESTful APIs. 
+active: developing-a-restful-api-with-ballerina
+intro: This simple guide helps you understand the basics of Ballerina constructs which allow you to write RESTful APIs. 
+redirect_from:
+  - /learn/developing-a-restful-api-with-ballerina
```

Comment:
Do we need the redirect from here?

Reply:
We need to add the entry without the slash.